rebar.config

```diff
@@ -25,7 +25,7 @@
   {telemetry, "0.4.1"},
   {psql_migration, {git, "https://github.com/helium/psql-migration.git", {branch, "master"}}},
   {clique, {git, "https://github.com/helium/clique.git", {branch, "develop"}}},
-  {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "master"}}},
+  {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "rg/data-credits"}}},
```

Comment:
Fix this before merge

Reply:
Yeah, once core branch lands will do it :+1: